test/e2e-kind.sh

```diff
@@ -22,13 +23,17 @@ run_ct_container() {
         teamcity_volume=(-v /teamcity/system/git:/teamcity/system/git)
     fi
     docker run --rm --interactive --detach --network host --name ct \
-        --volume "$(pwd)/test/ct-e2e.yaml:/etc/ct/ct.yaml" \
-        --volume "$(pwd):/workdir" \
         "${teamcity_volume[@]}" \
-        --workdir /workdir \
+        --workdir /charts \
         "quay.io/helmpack/chart-testing:${CT_VERSION}" \
         cat
     echo
+
+    docker cp "$(pwd)/test/ct-e2e.yaml" "ct:/etc/ct/ct.yaml"
+    docker cp "$(pwd)/." "ct:/charts"
+    docker_exec chown -R root:root /charts
+    # TODO(dlipovetsky) Debugging
```

Comment:
Do you want to remove that before merging?

Reply:
:facepalm: Thanks, will fix!